lib/Twig/Extension/Core.php

```diff
@@ -1152,9 +1152,23 @@ function _twig_escape_js_callback($matches)
 {
     $char = $matches[0];
 
-    // \xHH
-    if (!isset($char[1])) {
-        return '\\x'.strtoupper(substr('00'.bin2hex($char), -2));
+    /*
+     * A few characters have short escape sequences in JSON and JavaScript.
+     * Escape sequences supported only by JavaScript, not JSON, are ommitted.
+     * \" is also supported but omitted, because the resulting string is not HTML safe.
```

Comment:
Do you mean `\"` or `"` here?

Reply:
Both, sort of :-)  I mean that the escape sequence for the character `"` is `\"`. Neither the raw character nor the escaped string is HTML safe.